Sources/UserDefaultsGeneratorCore/ConfigurationParser.swift

```diff
@@ -0,0 +1,45 @@
+import Foundation
+import Yams
+
+public enum SwiftType: String, Decodable, Comparable {
+    public static func < (lhs: SwiftType, rhs: SwiftType) -> Bool {
+        return lhs.rawValue < rhs.rawValue
+    }
+    
+    case Int
```

Comment:
Should be lower case.

Reply:
Yes! I think so, too. I will fix it.

Reply:
:done:

Reply:
I decide to does not fix this suggest. 

https://github.com/bannzai/UserDefaultsGenerator/pull/3#issuecomment-532813197